index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prem Chand's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer"
+  />
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+</head>
+<body>
+    <header class = "pb-2">
+        <nav class = "pt-3 pb-3">
+            <ul class = "unordered-list d-flex flex-row justify-content-between">
+                <li class = "my-logo-line mt-1"><a href="">My Logo</a></li>
+                <li><img src="Images/hamburger.png" alt="hamburger-menu" class = "hamburger-height-width"></li>
+            </ul>
+        </nav>
+    </header>
+    
+    <main class = "pb-2 pt-5">
+        <section class = "p-3">
+            <h1 class = "name-tag">I'm Prem Chand <br> Glad to see <br> you!</h1>
+            <p class = "bio-info">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you 
+                like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <aside>
+                <h6><a href="">LET'S CONNECT</a></h6>
```

Comment:
- [ ] Upon review, I see that during the previous review, your font styles followed the design but after the update they are changed. The font size, weight, and line height should closely match the design for a consistent and visually appealing presentation.

| Your | Design |
| --- | --- |
| ![image](https://github.com/PremChand12/mobile-first-project/assets/55840999/e40753d7-76ea-4564-85ac-3acd5cbac65a) | ![image](https://github.com/PremChand12/mobile-first-project/assets/55840999/1ec23a4d-ae7f-434f-898c-777380b0189d) |

Please carefully review the design guidelines and make the necessary adjustments to your code to ensure that the font styles align with the specified requirements. Paying attention to these details will help create a cohesive and professional look for your project.